src/main/java/io/appium/java_client/Setting.java

```diff
@@ -23,10 +23,10 @@
 
     IGNORE_UNIMPORTANT_VIEWS("ignoreUnimportantViews"),
     WAIT_FOR_IDLE_TIMEOUT("waitForIdleTimeout"),
-    WAIT_FOR_SELECTOR_TIMEOUT("setWaitForSelectorTimeout"),
-    WAIT_SCROLL_ACKNOWLEDGMENT_TIMEOUT("setScrollAcknowledgmentTimeout"),
-    WAIT_ACTION_ACKNOWLEDGMENT_TIMEOUT("setActionAcknowledgmentTimeout"),
-    KEY_INJECTION_DELAY("setKeyInjectionDelay"),
+    WAIT_FOR_SELECTOR_TIMEOUT("waitForSelectorTimeout"),
+    WAIT_SCROLL_ACKNOWLEDGMENT_TIMEOUT("scrollAcknowledgmentTimeout"),
+    WAIT_ACTION_ACKNOWLEDGMENT_TIMEOUT("actionAcknowledgmentTimeout"),
+    KEY_INJECTION_DELAY("keyInjectionDelay"),
```

Comment:
Is this changed intended?

Reply:
yes @SrinivasanTarget  its intended

Reply:
OK I could see the changes [here](https://github.com/appium/appium-uiautomator2-server/pull/138/files)